docs/api/version-history.md

```diff
@@ -26,6 +27,8 @@ keywords: "API, Docker, rcli, REST, documentation"
   `net.ipv4.config.IFNAME.log_martians=1`. In API versions up-to 1.46, top level
   `--sysctl` settings for `eth0` will be migrated to `DriverOpts` when possible. 
   This automatic migration will be removed for API versions 1.47 and greater.
+
+* `POST /services/create` now supports OomScoreAdj
```

Comment:
Can you mention the update endpoint as well?

Reply:
The update endpoint was not included as part of this PR only create

Reply:
I think the same type is used for updating services as well; or at least, I think any property that can be set when creating a service must also be able to be changed (updateable) for the service (and swarmkit will reconcile)

Reply:
I think @thaJeztah has a point. Look at the most recent PR to extend the `POST /services/create` endpoint prior to this one: https://github.com/moby/moby/pull/46386. It affects both create and update. IIRC, when a Swarm service is updated, the Swarm scheduler reconciles by replacing all running tasks with new tasks that follow the updated task template in the service spec.